pages/privacy-policy.vue

```diff
@@ -0,0 +1,95 @@
+      </p>
+
+      <p>We are committed to protecting your&nbsp;privacy. We understand that you do not want the data you provide to GW2Bot to be distributed indiscriminately and this privacy policy explains how we collect data, what we do with it and what controls you have over&nbsp;it. All of the data we acquire from you is collected and processed in accordance with General Data Protection Regulations&nbsp;(GDPR). <strong>We don&rsquo;t share or sell any of your data to anyone and we&nbsp;never&nbsp;will.</strong></p>
+
+      <p>By using GW2Bot, you consent to the collection of data about your usage of GW2Bot in accordance with this privacy&nbsp;policy. If you use GW2Bot features that require authentication via <dfn><abbr title="Guild Wars 2">GW2</abbr></dfn> access token <em>(hereafter referred to as an&nbsp;&ldquo;<abbr>API</abbr>&nbsp;key&rdquo;),</em> then you also consent to the collection and use of your <abbr>GW2</abbr>&nbsp;gaming data in accordance with this privacy&nbsp;policy. If you do not agree with the use of your data as described in this privacy policy, please do not use&nbsp;GW2Bot. If you do not agree with the use of your <abbr>GW2</abbr> gaming data as described in this privacy policy, please do not use GW2Bot features that require authentication. <strong>We do not collect any personal data about you such as your name, your age, pictures of your cat, or any other personally identifiable&nbsp;information.</strong></p>
+
+      <p>GW2Bot requires access to Discord message content to enable <a class="external" href="https://wiki.guildwars2.com/wiki/Chat_link_format" target="_blank"><abbr>GW2</abbr> chat link</a> functionality within&nbsp;Discord. This means that GW2Bot can see your messages, but it cannot <em>&ldquo;read&rdquo;</em> or understand your messages; GW2Bot only searches new messages for a&nbsp;valid <abbr>GW2</abbr> chat link and then responds with the appropriate <abbr>GW2</abbr> game element if one is found, otherwise the message is disregarded. You can disable this feature by using the <code>/server preview_chat_links</code> command. <strong>GW2Bot does not record your message content and never&nbsp;will.</strong></p>
+
+      <p>We reserve the right to change this privacy policy from time to&nbsp;time. If we make any changes to this privacy policy, we will notify you by means of an announcement on the <a class="external" :href="$supportServerLink" target="_blank">GW2Bot Discord Server</a> prior to the changes becoming effective, which will give you the opportunity to review the changes before you choose to continue using&nbsp;GW2Bot. You will also receive a direct message about the changes from GW2Bot when you first interact with GW2Bot after this privacy policy is&nbsp;updated. You can always find a link to this privacy policy in the footer of every page on this website, or by using GW2Bot&rsquo;s <code>/help</code>&nbsp;command.</p>
+
+      <h2 id="what-data-do-we-collect" class="anchor">
```

Comment:
What are these id's for? Maybe you want these to be anchor tags instead of header tags?

Reply:
There are anchor tags within the text that link to the headings, for example:
https://gw2bot.info/privacy-policy#what-are-your-data-protection-rights
scrolls down to that section of the page